dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -292,9 +293,32 @@ public <T> T runChildTask(Task<T> task) throws MetadataDumperUsageException {
                         + "* Metadata has been saved to " + outputFile + "\n"
                         + "**********************************************************"
                 );
+            } else if (connector instanceof AbstractSummaryHolderJdbcConnector) {
+                AbstractSummaryHolderJdbcConnector.Summary summary =
+                    ((AbstractSummaryHolderJdbcConnector) connector).getSummary();
+                long totalRows = summary.rowCount();
+                if (totalRows > 0) {
+                    System.out.println(
+                        "**********************************************************\n"
+                            + "* Logs has been saved to " + outputFile + "\n"
+                            + "* Earliest Row Found at " + summary.earliestInterval().getStart()
+                            + "\n"
+                            + "* Latest Row Found at " + summary.latestInterval().getEndInclusive()
+                            + "\n"
+                            + "* Total Number of Rows Dumped " + totalRows + "\n"
+                            + "**********************************************************"
+                    );
+                } else {
+                    System.out.println(
+                        "**********************************************************\n"
+                            + "* No Rows were Found\n"
```

Comment:
Few suggestions for the case where there are no logs:

- does it make sense to log here a range, i.e `No rows were found in the range xxx - yyy`?
- we maybe want to emphasize this is a warning and prepend `WARNING` to the message
- if there are no logs dumped, the output of the connector is essentially useless. How about leaving a suggestion to increase dumping time range in the message?